server/tests/models/group.test.js

```diff
@@ -7,6 +7,11 @@ describe('Groups', () => {
   const subscriberObjectId = '57e590a0140ebf1cc48bb1bf';
 
   const userObjectId = '67e590a0140ebf1cc48bb1bf';
+  let group = null;
+
+  beforeEach(() => {
+    Group.remove({}, () => {});
```

Comment:
Is this needed with the test changes that @paradime made? The DB should be dropped after each test now